submission/launch.sh

```diff
@@ -4,16 +4,25 @@ cd "$( dirname "${BASH_SOURCE[0]}" )"
 
 trap "vagrant destroy -f" EXIT
 
-curl "${ARCHIVE_URL}" -o submission.zip
-curl "${SCRIPT_URL}" -o checker.sh
+curl "${VMCK_ARCHIVE_URL}" -o archive.zip
+curl "${VMCK_SCRIPT_URL}" -o script.sh
+
+touch reuslt.err
 
 vagrant up
-vagrant ssh -- < checker.sh > result.out
+(
+set +e
+vagrant ssh -- 'cd /vagrant; ./script.sh' 1> result.out 2> result.err
+)
+exit_code=$?
```

Comment:
Yeah the `)` above probably erases the error code. You should probably write it out to a file (i know...)